chibuezeayogu/React-Redux
# #161682143 Test Refactor

**What does the PR do?**
- Restructure test folder structure and refactors tests

**Description of Task to be completed?**
-  move test to correct test folder `__tests__`
- refactor tests
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `chore-implement-TTL-feedback-161682143`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161682143](https://www.pivotaltracker.com/story/show/161682143)

**Screenshots**
- N/A
